src/plugins/intel_gpu/src/plugin/ops/non_zero.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_gpu/plugin/program.hpp"
+#include "intel_gpu/plugin/common_utils.hpp"
+
+#include "ngraph/op/non_zero.hpp"
+
+#include "intel_gpu/primitives/non_zero.hpp"
+
+namespace ov {
+namespace runtime {
+namespace intel_gpu {
+
+static void CreateNonZeroOp(Program& p, const std::shared_ptr<ngraph::Node>& op) {
+    p.ValidateInputs(op, {1});
+    auto inputPrimitives = p.GetInputPrimitiveIDs(op);
+    std::string layerName = layer_type_name_ID(op);
+
+    cldnn::primitive_id countID = layerName + "_count";
+    auto primitive1 = cldnn::count_nonzero(countID,
+                                           inputPrimitives[0],
+                                           op->get_friendly_name());
+
+    // cldnn::primitive_id gatherID = layerName + "_gather";
```

Comment:
please remove unused codes

Reply:
I removed the unnecessary comments. Thank you!